recipes/openssl/1.x.x/conanfile.py

```diff
@@ -169,15 +169,9 @@ def _settings_build(self):
     def _full_version(self):
         return OpenSSLVersion(self.version)
 
-    @property
-    def _win_bash(self):
-        return self._settings_build.os == "Windows" and \
-               not self._use_nmake and \
-               (self._is_mingw or cross_building(self, skip_x64_x86=True))
-
     def export_sources(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+            copy(self, patch["patch_file"], self.recipe_folder, self.export_sources_folder )
```

Comment:
Maybe we could simplify a bit here using:

```
    def export_sources(self):
        export_conandata_patches(self)
```

Reply:
Hi @czoido,
Yes, nice simplification